### PR TITLE
switched to MUI icons && aligned them

### PR DESCRIPTION
**Pull request by JasperTheDog**
## Changes

Swapped from using fortawesome icons to Material UI Icons.
Made the necessary import changes.
Added stack tag formatting for alignment.

## Screenshots
1) Before
before

1) After
after

After Version 2 (Switched slideshow to copresent)
image

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [ ] All commits are tagged with the ticket number
- [x] No linting errors / newline at end of file warnings
- [x] All code follows repository-configured prettier formatting
- [x] No merge conflicts
- [x] All checks passing
- [x] Screenshots of UI changes (see Screenshots section)
- [x] Remove any non-applicable sections of this template
- [x] Assign the PR to yourself
- [x] No `yarn.lock` changes (unless dependencies have changed)
- [x] Request reviewers & ping on Slack
- [ ] PR is linked to the ticket (fill in the closes line below)

Closes # (issue #278

### CODE REVIEW

File: src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/ProjectDetails.tsx [hunk 1]
```diff
@@ -3,21 +3,25 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
-import { faFilePowerpoint, faFolderOpen, faList, faListOl } from '@fortawesome/free-solid-svg-icons';
+//import { faFilePowerpoint, faFolderOpen, faList, faListOl } from '@fortawesome/free-solid-svg-icons';
```
Comment (ryanhowe28) [inline src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/ProjectDetails.tsx:6]: remove commented line

**Review by ryanhowe28 — CHANGES_REQUESTED** [review]

**Review by RChandler234 — CHANGES_REQUESTED** [review]
Can you use the CoPresent Icon for the slide deck instead of the current one? Should be good to merge after that change

**Review by RChandler234 — APPROVED** [review]
Looks good to me!

**Review by ryanhowe28 — APPROVED** [review]

**Review by ryanhowe28 — COMMENTED** [review]
Looks great!

**Review by anthonybernardi — APPROVED** [review]
nice work!!

**Pull request merged**